source/blog/2018-04-06-test-day-for-fedora-28-atomic-cloud-april-11th.html.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Test Day for Fedora 28 Atomic/Cloud April 11th
+author: sinnykumari
+date: 2018-04-06
+comments: true
+published: true
+tags: atomic, fedora, test_day, cloud
+---
+
+With the Fedora 28 Beta [official release announcement](https://fedoramagazine.org/announcing-fedora-28-beta/), the [Fedora Atomic Working Group](https://fedoraproject.org/wiki/Atomic_WG) and [Fedora Cloud SIG](https://fedoraproject.org/wiki/Cloud_SIG) want some love from the community to find and squash bugs. We are organizing a Test Day on Wednesday, April 11th 2018.
```

Comment:
nit:
> want some love from the community

accidentally sounds negative to me as if the community doesn't usually love us. Maybe "would like to invite members of the community to join us in finding and squashing bugs" or something like that.

Reply:
Thanks @ashcrow for the review.
It didn't cross my mind that it could be taken in negative way :) Updated PR with required changes!